app/services/current_team_and_user/sar.rb

```diff
@@ -0,0 +1,4 @@
+module CurrentTeamAndUser
+  class SAR < Base
```

Comment:
Use the full path/name of `Base` -> `CurrentTeamAndUser::Base`. Ruby doesn't fix the issue of trying to resolve class names in the top namespace until 2.5, which I think could cause a side-effect here.